packages/components/radio/src/less/_mixins.less

```diff
@@ -123,6 +131,18 @@
     .@{selector}__description {
       padding-left: @icon-size + @oui-radio-padding-left;
     }
+
+    .@{selector}__footer {
+      background: @oui-choice-thumbnail-footer-background;
```

Comment:
We should avoid using variables from an other component: `@oui-choice-*` -> `@oui-radio-*`.